In [1]:
import pandas as pd

In [ ]:
titanic = pd.read_csv(r'https://raw.githubusercontent.com/lucchesia7/coding-temple-da-sql-day-1/main/day_1/data/titanic.csv')
titanic.head()

In [3]:
titanic.columns = titanic.columns.str.lower().str.replace(' ', '_').str.replace('/', '_')

In [4]:
connection = 'postgresql://nmqlktgb:gwvXnF-bavIXr-Io88sZBC4_63_W6vE9@peanut.db.elephantsql.com/nmqlktgb'

In [5]:
titanic.to_sql('titanic', con = connection, if_exists = 'fail')

887

Question 1: Count how many rows you have.

```sql
SELECT COUNT(*) FROM titanic
```

Answer: 887






Question 2: How many people survived?

```sql
SELECT SUM(survived) FROM titanic
```

Answer: 342

Question 3: What passenger class has the largest population?

```sql
SELECT pclass, COUNT(pclass) AS pclass_count FROM count
GROUP BY pclass
ORDER BY pclass_count
DESC
LIMIT 1
```

Answer: Pclass 3 had 487 passengers in it

Bonus: Give me all the passengers that had their name start with 'Miss'

```sql
SELECT * FROM titanic
WHERE name LIKE 'MISS%'
```